client/src/components/Header/NavBar/NavBar.tsx

```diff
@@ -29,55 +31,52 @@ const NavList = styled.ul`
   list-style: none;
   display: flex;
   align-items: center;
-  font-size: 12px;
+  font-size: ${(props) => props.theme.fontSize.tiny};
 `;
 
 const NavListItem = styled.li`
-  color: #717171;
-
-  a {
-    text-decoration: none;
-    color: inherit;
-    padding: 0 10px;
-  }
-
-  .seperator {
-    display: inline-block;
-    width: 1px;
-    height: 7px;
-    background-color: #ddd;
-  }
+  color: ${(props) => props.theme.color.grey4};
+  padding: 0 10px;
+`;
 
-  :last-child .seperator {
-    display: none;
-  }
+const Badge = styled.span`
+  font-size: 10px;
+  line-height: 11px;
+  color: ${(props) => props.theme.color.mint2};
+  padding: 0 2px;
+`;
 
-  .badge {
-    font-size: 10px;
-    line-height: 11px;
-    color: #29c1bc;
-    padding: 0 2px;
-  }
+type SeperatorProps = {
+  isLastItem: boolean;
+};
+const Seperator = styled.span<SeperatorProps>`
+  display: ${(props) => (props.isLastItem ? 'none' : '')};
+  width: 1px;
+  height: 7px;
+  background-color: ${(props) => props.theme.color.grey2};
```

Comment:
스타일적인 부분에서의 lastChild라면 이렇게 사용할 수도 있겠네요 !! 

```suggestion
const Seperator = styled.span`
  display: block;
  width: 1px;
  height: 7px;
  background-color: ${(props) => props.theme.color.grey2};
  &:last-child {
    display: none;
  }
```
이렇게 하게 되면 props 사용이 줄어들 것 같습니다 :)

Reply:
앗 이미 논의된 내용이였군요 ! css에 정의하게 되면 확인이 힘들어서 props로 전달하는 느낌으로 하셨던 거였네요 ! 👍

Reply:
위에 진우님 리뷰에도 답변 달아뒀는데 이렇게 하면 컴포넌트 return 부에서는 마지막 `Seperator`는 랜더링 되지 않는 것을 알기가 어려운 것 같아서요.. ㅎㅎ 참고하겠습니다. 감사합니다. 🙂